pkg/clustermanager/cluster_manager.go

```diff
@@ -961,3 +996,19 @@ func (c *ClusterManager) bundlesFetcher(cluster *types.Cluster) cluster.BundlesF
 		return c.clusterClient.GetBundles(ctx, cluster.KubeconfigFile, name, namespace)
 	}
 }
+
+func (c *ClusterManager) DeleteGitOpsConfig(ctx context.Context, managementCluster *types.Cluster, name string, namespace string) error {
+	return c.clusterClient.DeleteGitOpsConfig(ctx, managementCluster, name, namespace)
+}
+
+func (c *ClusterManager) DeleteOIDCConfig(ctx context.Context, managementCluster *types.Cluster, name string, namespace string) error {
+	return c.clusterClient.DeleteOIDCConfig(ctx, managementCluster, name, namespace)
+}
+
+func (c *ClusterManager) DeleteAWSIamConfig(ctx context.Context, managementCluster *types.Cluster, name string, namespace string) error {
+	return c.clusterClient.DeleteOIDCConfig(ctx, managementCluster, name, namespace)
```

Comment:
this should probably be delete AWSIAM config

Reply:
done